docs/get-started/bridging.mdx

```diff
@@ -26,12 +26,9 @@ The Etherlink indexer run by Nomadic Labs automatically executes these bridging
 
 ## Bridging your tokens
 
-Etherlink provides bridges for tez tokens:
+Etherlink provides a [Mainnet canonical bridge](https://bridge.etherlink.com/) for tez tokens.
```

Comment:
Maybe mention that the bridge is trustless and permissionless (no third party involved, everyone can use it without any restrictions)

Reply:
> This bridge is trustless and permissionless; anyone can use it without restrictions or the intervention of a third party.